stage0/run.go

```diff
@@ -466,5 +468,5 @@ func launchMetadataService(debug bool) error {
 		Stderr: os.Stderr,
 	}
 
-	return cmd.Start()
+	return cmd.Run()
```

Comment:
@eyakubovich Hmm, if the parent exits, then the metadata service will just be taken care by the PID 1, right? 
>
We can do this but it'll avoid a zombie only in some scenarios. First there's a race -- only works if the main goroutine exits after metadata service has exited and this goroutine collected status. And if there're multiple pods, metadata service will exit when the last pod unregisters. If the last pod is not the one that launched the metadata service, it won't help as well.